### DISCUSSION TITLE
Tables - top alignment

### DISCUSSION THREAD

**Discussion by sregier206** [GitbookIO/community#824] [0. General bugs and questions] [answered]
Currently, you offer left, right, and center alignment in a cell. 

It is also important to top-align text (or images, etc), similar to a cell in Excel. Visually, this helps the eye follow the row, especially with multiple rows. Can we have this added to GitBooks?

**Comment by viktorrenkema** [comment]
Interesting suggestion, I happen to be working on improvements to tables so i'll add it to my list to look into it!

**Comment by JaniceManwiller** [comment]
I requested this over a year ago - https://github.com/orgs/GitbookIO/discussions/272.

It is very difficult to read multi-line text across a row when the top line of the text jumps around.

**Comment by sregier206** [comment]
Agreed, horizontal and vertical alignment options in tables should be standard and are usually expected in tables., not just spreadsheet-type software. 

It is also an accessibility issue, as people with vision issues who need to zoom their browser window size past 100% can have a hard time following a row if every cell in it is a different height, see [W3 guidelines](https://www.w3.org/WAI/tutorials/tables/tips/#:~:text=Alignment%3A%20Align%20text%20to%20the%20left%20and%20numeric,if%20a%20cell%20spans%20more%20than%20one%20column.) for an example of a recommended standard and reference. 

Different cell heights across the same row in a table  also adds additional eye strain.

**Reply by viktorrenkema** [reply, marked as answer]
Hey @sregier206 and @JaniceManwiller , I just released the ability to set vertical alignment on text columns. You can pick the vertical alignment by clicking on the column options, within the dropdown you should be able to make your pick.

Let me know how this works for you, and if you have any feedback :)

**Reply by JaniceManwiller** [reply]
Thank you so much! Just tried it out on a smaller space, and looks like it works. So much easier to read!

One suggestion - it's unlikely that I would want different vertical alignment in different columns. It would be nice to have a table-wide setting for that can be overridden per-column if needed.

**Reply by viktorrenkema** [reply]
This did cross my mind and may be something we'll also do later on! For now it was most impactful to release on per-column basis, because 1. some use-cases would require differing alignments per-column, and having the setting affect _all_ cells would fully block those use-cases, 2. the number of tables that have 10+ columns where it's too tedious of a job to do it one by one is pretty small and 3. we could allow both but this would introduce complexity and the UX would require more time and just generally it's worthwhile to spend time on another feature request from you or others in the community :)

But it's def. on my radar and even more so as you also mentioned it now!

**Comment by sregier206** [comment]
Fantastic @viktorrenkema thank you SOOOOOO MUCH!
